test/functional/ui/syntax_conceal_spec.lua

```diff
@@ -0,0 +1,402 @@
+          ~                                                    |
+          ~                                                    |
+          ~                                                    |
+          ~                                                    |
+          :syn match dAmpersand '[&][&]' conceal cchar=∧       |
+        ]])
+      end)
+
+      it('double ampersands and then move the cursor to the beginning of the file and back to the end of the file.', function() 
+        feed("ggG")
+        screen:expect([[
+          {1:∧}                                                    |
+          {1:∧}                                                    |
+          {1:∧}                                                    |
+          {1:∧}                                                    |
+          {1:∧}                                                    |
+          {1:∧}                                                    |
+          ^                                                     |
+          ~                                                    |
+          ~                                                    |
+          ~                                                    |
+          ~                                                    |
+          ~                                                    |
+          ~                                                    |
+          :syn match dAmpersand '[&][&]' conceal cchar=∧       |
+        ]])
+      end)
+    end) -- Match 
+      
+    it("keyword instances in initially in the document.", function()
+      feed("2ilambda<cr><ESC>")
+      execute("let &conceallevel=1")
+      execute("syn keyword kLambda lambda conceal cchar=λ")
+      screen:expect([[
+        {1:λ}                                                    |
+        {1:λ}                                                    |
+        ^                                                     |
+        ~                                                    |
+        ~                                                    |
+        ~                                                    |
+        ~                                                    |
+        ~                                                    |
+        ~                                                    |
+        ~                                                    |
+        ~                                                    |
+        ~                                                    |
+        ~                                                    |
+        :syn keyword kLambda lambda conceal cchar=λ          |
+      ]])
+    end) -- Keyword
+
+    describe("regions in the document", function()
+
+      before_each(function() 
+        feed("2")
+        insert("<r> a region of text </r>\n")
+        execute("let &conceallevel=1")
+      end)
+      
+      describe('initially', function()
```

Comment:
I think this describe block is a bit redundant, you can just put "initially" first in the 2  `it` descriptions it contains.